sites/fast-website/src/public/index.html

```diff
@@ -131,7 +131,160 @@ <h3>AngularJS (1.x) 1.7.9</h3>
+									<fast-anchor
+											slot="footer"
+											class="link"
+											href="#"
+											appearance="lightweight"
+											>View Github</fast-anchor
+									>
+									<fast-anchor
+											slot="footer"
+											class="link"
+											href="#"
+											appearance="lightweight"
+											>Read Documentation</fast-anchor
+									>
+							</feature-card>
+							<feature-card>
+									<h5 slot="header">05</h5>
+									<h4 slot="header">Animation</h4>
+									<p>Design sophisticated animation sequences with the animation library, an interface for the Web Animations API.</p>
+									<fast-anchor
+											slot="footer"
+											class="link"
+											href="#"
+											appearance="lightweight"
+											>View Github</fast-anchor
+									>
+									<fast-anchor
+											slot="footer"
+											class="link"
+											href="#"
+											appearance="lightweight"
+											>Read Documentation</fast-anchor
+									>
+							</feature-card>
+							<feature-card>
+									<h5 slot="header">06</h5>
+									<h4 slot="header">Colors</h4>
+									<p>Create color palettes, extract colors from images, and handle other color operations using our color library.</p>
+									<fast-anchor
+											slot="footer"
+											class="link"
+											href="#"
+											appearance="lightweight"
+											>View Github</fast-anchor
+									>
+									<fast-anchor
+											slot="footer"
+											class="link"
+											href="#"
+											appearance="lightweight"
+											>Read Documentation</fast-anchor
+									>
+							</feature-card>
+							<feature-card>
+									<h5 slot="header">07</h5>
+									<h4 slot="header">Layout / Grid</h4>
+									<p>Build layouts such as a 12 column grid for content or an application grid with resizable panels.</p>
+									<fast-anchor
+											slot="footer"
+											class="link"
+											href="#"
+											appearance="lightweight"
+											>View Github</fast-anchor
+									>
+									<fast-anchor
+											slot="footer"
+											class="link"
+											href="#"
+											appearance="lightweight"
+											>Read Documentation</fast-anchor
+									>
+							</feature-card>
+							<feature-card>
+									<h5 slot="header">08</h5>
+									<h4 slot="header">Other</h4>
+									<p>Leverage a toolkit of general utilities such as keyboarding, Right-To-Left (RTL), number, and string manipulation.</p>
+									<fast-anchor
+											slot="footer"
+											class="link"
+											href="#"
+											appearance="lightweight"
+											>View Github</fast-anchor
+									>
+									<fast-anchor
+											slot="footer"
+											class="link"
+											href="#"
+											appearance="lightweight"
+											>Read Documentation</fast-anchor
+									>
+							</feature-card>
+						</card-section>
```

Comment:
Yikes - can we fix this formatting?

Reply:
Aaron pointed the same thing out.  Local file looks fine.  If there is something I can change on my end to fix that I'd love to do so, I'm not sure what it would be though.

Reply:
@freefaller69 It looks like these are tabs instead of spaces.

Reply:
Let me see what I can do.

Reply:
I think I got it.  My editor was setting tabs instead of spaces to indent on return even though I don't have it set to do so as far as I could find.